docs/sync.js

```diff
@@ -139,15 +140,17 @@ class User {
 	static login(server, username, password, callback) {}
 
 	/**
-	 * Login a sync user using an external login provider.
+	 * Register/login a sync user using an external login provider.
 	 * @param {string} server - authentication server
-	 * @param {string} provider - The provider type
-	 * @param {string} providerToken - The access token for the given provider
+   * @param {object} options - options, containing the following:
```

Comment:
It is! Even after I fixed it. It looks like the file actually has tabs rather than spaces. I'll fix the whole thing.